bbpower/power_summarizer.py

```diff
@@ -421,6 +581,11 @@ def run(self):
         summ['saccs'][3].save_fits(self.get_output("cells_null"),
                                    overwrite=True)
 
+        print('Saving mean sims spectra')
```

Comment:
Great idea. We could consider making saving the mean spectra optional, but since this wouldn't use up a lot of space I'm all for keeping it.